src/main/java/com/ericsson/ei/controller/SubscriptionController.java

```diff
@@ -51,21 +54,30 @@
         List<com.ericsson.ei.controller.model.Subscription> subscription);
 
     /**
-     * This method returns the subscriptions for the given subscription names.
+     * This method removes subscriptions from the database. It requires a list of names by using the 'names' parameter
      * 
      */
-    @RequestMapping(value = "/{subscriptionNames}", method = RequestMethod.GET)
-    public ResponseEntity<?> getSubscriptionByNames(
+    @RequestMapping(value = "", method = RequestMethod.DELETE)
+    public ResponseEntity<?> deleteSubscriptions(
+        @RequestParam
+        String subscriptionName);
```

Comment:
See my previous comment about the parameter name

Reply:
So after talking more about this the parameter is now called subscriptionNames in plural.